src/main/de/linearbits/preferences/EditorBoolean.java

```diff
@@ -28,31 +28,43 @@
     /** Widget*/
```

Comment:
Whole class: various issues with intendations and blanks. Code looks fina apart from formatting.